client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -432,45 +434,46 @@ export class BpmnEditor extends CachedComponent {
 
     const selectionLength = selection.get().length;
 
-    const inputActive = isInputActive();
```

Comment:
This is breaking for all plugins relying on inputActive. I'd say let's leave the state property as is, but the shortcuts don't need to rely on it.

Reply:
Also, as just discovered with @jarekdanielak, the `getEditMenu` still depends on inputActive.

Reply:
Isn't backwards compatibility covered via https://github.com/camunda/camunda-modeler/pull/4620/files/725723408e34c74c5ab7ccb2cefb2cf3b2f2d766#diff-8d663070a9293c2317ca1a7dff4fa8a7004cbd6855d29d128635aad4fa20d67cR475?